docs/static/running-logstash-command-line.asciidoc

```diff
@@ -68,6 +68,22 @@ With this command, Logstash concatenates three config files, `/tmp/one`, `/tmp/t
   If you wish to use both defaults, please use the empty string for the `-e` flag.
   The default is nil.
 
+*`--modules`*::
+  Launch the named module.  Works in conjunction with the `-M` option to assign values to
+  default variables for the specified module.  If `--modules` is used on the command line,
+  any modules in `logstash.yml` will be ignored, as will any settings there.  This flag is
+  mutually exclusive to the `-f` and `-e` flags.  Only one of `-f`, `-e`, or `--modules` may
+  be specified.  Multiple modules can be specified by separating them with a comma, or by
+  invoking the `--modules` flag multiple times.
```

Comment:
I don't think so.  Modules should be exclusive of other pipelines.  @suyograo, that's what you wanted, at least.  We can change it later if such is wanted.